client/src/app/actions/apiRoute.js

```diff
@@ -0,0 +1,9 @@
+//For production
+// const ApiRoute = '/api/v1'
+
+// export default ApiRoute
+
+
+const ApiRoute = 'http://localhost:5000/api/v1'
```

Comment:
no need to have this: `http://localhost:5000` it can simply be `api/v1`

Reply:
i am routing to a server on another port different from my client side. I have redefined it in my webpack so this has been deleted.